src/ui/component/common/markdown-preview-internal.jsx

```diff
@@ -31,19 +43,53 @@ const schema = { ...defaultSchema };
 
 // Extend sanitation schema to support lbry protocol
 schema.protocols.href.push('lbry');
+schema.attributes.a.push('data-preview');
 
 const MarkdownPreview = (props: MarkdownProps) => {
-  const { content, promptLinks } = props;
-  const remarkOptions = {
+  const { content, strip, promptLinks } = props;
+
+  const remarkOptions: Object = {
     sanitize: schema,
+    fragment: React.Fragment,
     remarkReactComponents: {
       a: promptLinks ? ExternalLink : SimpleLink,
+      // Workaraund of remarkOptions.Fragment
+      div: React.Fragment,
     },
   };
+
+  const remarkAttrOpts = {
+    scope: 'extended',
+    elements: ['link'],
+    extend: { link: ['data-preview'] },
```

Comment:
We probably don't need to use the `data-` prefix since it doesn't get passed to the actual DOM,
and it will make the syntax more simple to use, what do you think ?
`[link](url){preview=false}` or `[link](url){embed=false}`